src/main/java/com/dev/cinema/controller/MovieController.java

```diff
@@ -27,7 +28,7 @@ public MovieController(MovieService movieService, MovieMapper movieMapper) {
     }
 
     @PostMapping
-    public void create(@RequestBody MovieRequestDto movieRequestDto) {
+    public void create(@RequestBody @Valid MovieRequestDto movieRequestDto) {
```

Comment:
what about MovieSessionController?

Reply:
hmmm, in my code i have it

Reply:
![image](https://user-images.githubusercontent.com/36328484/108349340-89ea0280-71eb-11eb-81f5-f05e72dc4338.png)

Reply:
MovieSessionController and MovieController are different controllers